zmoney/Services/ZService.swift

```diff
@@ -28,6 +28,7 @@ struct Zservice {
     public var isLoggedIn: Bool {
         !TokenService.shared.accessToken.isEmpty
     }
+    private var lastSyncTimeStamp = UserDefaults.standard.integer(forKey: "lastSyncTimeStamp")
```

Comment:
Здесь можно вот так сделать, чтобы логику работы с UserDefaults держать в одном месте

```suggestion
    private var lastSyncTimeStamp: Int {
        get {
            UserDefaults.standard.integer(forKey: "lastSyncTimeStamp")
        }
        set {
            UserDefaults.standard.set(newValue, forKey: "lastSyncTimeStamp")
        }
    }
```

Reply:
Насчет этого я не очень уверен - мне нужно делать метод mutating, а так как я это еще и в замыкании делаю - мне сыпет ошибкой

Reply:
Согласен, со struct не очень заходит. Тогда лучше вообще убрать свойство lastSyncTimeStamp, чтобы не было намека на Statefull сервис